docs/client/rest/rest_api.md

```diff
@@ -99,9 +99,10 @@ Create a session
 
 #### Response Body
 
-| Name       | Description                   | Type   |
-|:-----------|:------------------------------|:-------|
-| identifier | The session handle identifier | String |
+| Name           | Description                                                                                         | Type   |
+|:---------------|:----------------------------------------------------------------------------------------------------|:-------|
+| identifier     | The session handle identifier                                                                       | String |
+| kyuubiInstance | The Kyuubi instance that holds the session and to call for the following operations in the session. | String |
```

Comment:
Look at the end of most of the Description without a dot`.`, can it be removed here?

Reply:
thanks, fixed